src/app/app.component.html

```diff
@@ -38,10 +37,7 @@
           }
         </div>
         <div class="phone:hidden justify-end hidden md:flex font-medium w-[200px]">
-          <a href="#"
-             class="hover:bg-[#FFC907] hover:bg-opacity-20 transition-all duration-400 ease-in-out px-5 py-2.5 rounded-[40px]"
-             i18n
-          >
+          <a href="#" class="hover:bg-[#FFC907] hover:bg-opacity-20 transition-all duration-400 ease-in-out px-5 py-2.5 rounded-[40px]" i18n>
```

Comment:
Create own key on tailwindcss configuration for `hover:bg-[#FFC907]`

Reply:
done